client/src/main/com/sinch/sdk/domains/voice/adapters/ConferencesService.java

```diff
@@ -0,0 +1,54 @@
+package com.sinch.sdk.domains.voice.adapters;
+
+import com.sinch.sdk.core.http.AuthManager;
+import com.sinch.sdk.core.http.HttpClient;
+import com.sinch.sdk.core.http.HttpMapper;
+import com.sinch.sdk.domains.voice.adapters.api.v1.ConferencesApi;
+import com.sinch.sdk.domains.voice.adapters.converters.CalloutsDtoConverter;
+import com.sinch.sdk.domains.voice.adapters.converters.ConferencesDtoConverter;
+import com.sinch.sdk.domains.voice.models.requests.CalloutRequestParametersConference;
+import com.sinch.sdk.domains.voice.models.requests.ConferenceManageParticipantRequestParameters;
+import com.sinch.sdk.domains.voice.models.response.ConferenceParticipant;
+import com.sinch.sdk.models.Configuration;
+import java.util.Collection;
+import java.util.Map;
+
+public class ConferencesService implements com.sinch.sdk.domains.voice.ConferencesService {
+
+  private final ConferencesApi api;
+
+  public ConferencesService(
+      Configuration configuration, HttpClient httpClient, Map<String, AuthManager> authManagers) {
+    this.api =
+        new ConferencesApi(
+            httpClient, configuration.getVoiceServer(), authManagers, new HttpMapper());
+  }
+
+  protected ConferencesApi getApi() {
+    return this.api;
+  }
+
+  public String call(CalloutRequestParametersConference parameters) {
+    return CalloutsDtoConverter.convert(
+        getApi().callouts(CalloutsDtoConverter.convert(parameters)));
```

Comment:
What's the final word about the this? Do we need to expose the same service through 2 different paths?

Reply:
Not sure there was discussion about this point.
Because of documentation and OAS tags the service twice, I just added dedicated call to this service